main.py

```diff
@@ -1,7 +1,30 @@
 import init_django_orm  # noqa: F401
 
 from django.db.models import QuerySet
+from db.models import Genre, Actor
 
 
 def main() -> QuerySet:
-    pass
+    genres = ["Western", "Action", "Dramma"]
```

Comment:
The genre 'Dramma' seems to be a typo. It is later updated to 'Drama', but it would be better to correct it initially to avoid unnecessary updates.